lib/client/indexer/proposals.ts

```diff
@@ -237,13 +237,11 @@ export class FutarchyIndexerProposalsClient implements FutarchyProposalsClient {
               startSlot: p.initial_slot,
               endSlot: p.end_slot,
               // TODO figure this out by slot enqueued maybe
-              creationDate: p.created_at,
-              endDate:
-                p.ended_at ?? new Date(new Date(p.created_at).getDate() + 3),
+              // TODO used updated_at because created_at date always the same in the indexer
+              creationDate: new Date(p.updated_at),
```

Comment:
Creation date is actually the date that the proposal was created, I just filled them into the DB cause the indexer wasn't running. So it's valid and accurate to use created at vs updated at which is when the state changes. of which we could expect 3-4 update_at changes.